src/UglyToad.PdfPig/Writer/PdfMerger.cs

```diff
@@ -67,13 +69,13 @@ public static byte[] Merge(params string[] filePaths)
                 bytes.Add(File.ReadAllBytes(filePath));
             }
 
-            return Merge(bytes);
+            return Merge(bytes, null);
         }
 
         /// <summary>
         /// Merge the set of PDF documents.
         /// </summary>
-        public static byte[] Merge(IReadOnlyList<byte[]> files)
+        public static byte[] Merge(IReadOnlyList<byte[]> files, IReadOnlyList<IReadOnlyCollection<int>> pagesBundle = null)
```

Comment:
`IReadOnlyCollection` or `ICollection` are probably the wrong types here, since they do not guarantee order iirc, I'd recommend `IReadOnlyList` since order is important here.